build_debian.sh

```diff
@@ -43,6 +43,9 @@ HOSTNAME=sonic
 DEFAULT_USERINFO="Default admin user,,,"
 BUILD_TOOL_PATH=src/sonic-build-hooks/buildinfo
 TRUSTED_GPG_DIR=$BUILD_TOOL_PATH/trusted.gpg.d
+## Remapped docker usernames and group names
+REMAPREDIS=remapredis
```

Comment:
Question - should we use a meaningful name here? remapredis doesn't sound like something that is describing this item.
Maybe use unpriv-redis? Do depict what we intend this user to be.

Reply:
Yes, we can mutually agree on better variable name here